### PR TITLE
Update EIP-3540: Set prefix to 0xEF00

### PR DESCRIPTION
**Pull request by chfast**

### CODE REVIEW

File: EIPS/eip-3540.md [hunk 1]
```diff
@@ -170,6 +170,16 @@ Finally, create transaction must be allowed to contain legacy _initcode_ and dep
...
 The `0xEF` byte was chosen because it is reserved for this purpose by [EIP-3541](./eip-3541.md).
 
+### The MAGIC byte
+
+1. The `0x00` byte was chosen to avoid clashes with three contracts which were deployed on **Mainnet**:
+  - `EFF09f918bf09f9fa9` [0xca7bf67ab492b49806e24b6e2e4ec105183caa01](https://etherscan.io/address/0xca7bf67ab492b49806e24b6e2e4ec105183caa01)
+  - `EF` [0x897da0f23ccc5e939ec7a53032c5e80fd1a947ec](https://etherscan.io/address/0x897da0f23ccc5e939ec7a53032c5e80fd1a947ec)
+  - `EF` [0x6e51d4d9be52b623a3d3a2fa8d3c5e3e01175cd0](https://etherscan.io/address/0x6e51d4d9be52b623a3d3a2fa8d3c5e3e01175cd0)
+
+2. No contracts starting with `0xEF` bytes exist public testnets: Goerli, Ropsten, Rinkeby, Kovan and Sepolia at their London fork block.
```
Comment (gumb0) [inline EIPS/eip-3540.md:180]:
```suggestion
2. No contracts starting with `0xEF` bytes exist on public testnets: Goerli, Ropsten, Rinkeby, Kovan and Sepolia at their London fork block.
```

File: EIPS/eip-3540.md [hunk 2]
```diff
@@ -253,32 +261,168 @@ All cases should be checked for creation transaction, `CREATE` and `CREATE2`.
...
+  - Results don't differ when executed inside legacy or EOF contract
 
 ## Reference Implementation
 
 ### Generic Implementation
 
 ```python
 FORMAT = 0xEF
-MAGIC = 0x00 # To be defined
+MAGIC = 0x00
```
Comment (gumb0) [inline EIPS/eip-3540.md:289]: These two versions of the code that are being added are in fact outdated, should be removed.

**Review by gumb0 — CHANGES_REQUESTED** [review]

**Review by gumb0 — APPROVED** [review]

File: EIPS/eip-3540.md [hunk 3]
```diff
@@ -327,13 +335,17 @@ def validate_eof(code: bytes):
     assert(len(code) == (pos + section_sizes[S_CODE] + section_sizes[S_DATA]))
 ```
 
+- evmone: https://github.com/ethereum/evmone/pull/303
```
Comment (axic) [inline EIPS/eip-3540.md:338]: These aren't supposed to be here anymore.

**Review by axic — APPROVED** [review]

**Comment by alita-moore** [comment]
@axic did you close and reopen because you waited a few minutes and it wasn't responding or did you do that right after reviewing?

**Comment by axic** [comment]
A few minutes after, but not too long.

**Comment by alita-moore** [comment]
okay, I see the chfast also marked this as ready for review after your review. Do you think this was related?

**Comment by axic** [comment]
Likely that was the case.

**Pull request merged**